### PR TITLE
Allow for campaigns to be managed externally.

### PR DESCRIPTION
**Pull request by dwick**
:eyeglasses: @zeantsoi @tsegaran 

related to https://github.com/reddit/reddit-public/pull/5009

### CODE REVIEW

File: reddit_adzerk/adzerkpromote.py [hunk 1]
```diff
@@ -988,16 +991,21 @@ def adzerk_request(keywords, uid, num_placements=1, timeout=1.5,
         )
 
         if not campaign:
-            adzerk_campaign_id = decision['campaignId']
+            link_obj = Link._by_fullname(link, data=True, stale=True)
```
Comment (zeantsoi) [inline reddit_adzerk/adzerkpromote.py]: Dunno if it's within the scope of this PR, but can we change [`link` to `link_fullname`](https://github.com/reddit/reddit-plugin-adzerk/pull/166/files#diff-8ff7bdcffc2cdf19a06e2d539306378aR977) so its definition is less likely to be confused?

**Comment by zeantsoi** [comment]
One comment... otherwise LGTM :nail_care:

**Comment by dwick** [comment]
:haircut:

**Comment by zeantsoi** [comment]
:fish:

**Comment by tsegaran** [comment]
:fish:

**Pull request merged**